docs/source/specifications/projjson.rst

```diff
@@ -71,19 +119,123 @@ The high level objects are:
+
+An object can sometimes be identified in different ways, in which case the ``ids``
+member can be used to specify a JSON array of objects with the same type of ``id``.
+
+Identifiers are allowed in top-level objects and inner objects. The WKT2:2019
+specification recommends that if an object has an identifier, its inner objects
+should omit their identifiers, with the exceptions mentioned at
+http://docs.opengeospatial.org/is/18-010r7/18-010r7.html#37.
+
+Object usages
++++++++++++++
+
+CRS and coordinate operation objects are derived classes (in object modeling terminology)
+of a "object usage" class. An object usage has the following optional members:
+
+- ``scope``: (optional) value of type string describing the purpose or purposes of
+  the object. e.g "Geodesy, topographic mapping and cadastre"
+- ``area``: (optional) value of type string which describes a geographic area
+  over which a CRS or coordinate operation is applicable. e.g. "World"
+- ``bbox``: (optional) value of type object, with 4 required members:
+  * ``east_longitude``: (required) number expressing the longitude in degrees in [-180,180] range of the eastern most part of the extent.
+  * ``west_longitude``: (required) number expressing the longitude in degrees in [-180,180] range of the western most part of the extent. For an extent crossing the anti-meridian, west_longitude is lower than east_longitude.
+  * ``south_latitude``: (required) number expressing the latitude in degrees in [-90,90] range of the southern most part of the extent.
+  * ``north_latitude``: (required) number expressing the latitude in degrees in [-90,90] range of the northern most part of the extent.
+  The coordinates are expressed in a unspecified datum, with the longitudes
+  relative to the international reference meridian.
+- ``remarks``: (optional) value of type string with an informative text that does
+  not modify the definining parameters of the object. e.g "Use NTv2 file for better accuracy"
+- ``id`` (mutually exclusive with ``ids``): (optional) Identifier of the object, as defined in :ref:`identifiers`
+- ``ids`` (mutually exclusive with ``id``): (optional) Identifiers of the object, as defined in :ref:`identifiers`
+
+If several extents and scopes apply to an object, the ``scope``, ``area`` and ``bbox``
+members should not be used. Instead a ``usages`` member should be used, whose value
+is an array of objects, each of them accepting ``scope`` and/or ``area`` and/or ``bbox``
+as members. While it is acceptable to use the ``usages`` construct for a single
+usage, it is recommended to avoid it and rather use instead the
+``scope``, ``area`` and ``bbox`` members.
+
+Units
++++++
+
+A unit may be described either as:
+
+- an object with the following members:
+    * ``type``: (required) value of type string among the following enumeration:
+      ``LinearUnit``, ``AngularUnit``, ``ScaleUnit``, ``TimeUnit``, ``ParametricUnit``, ``Unit``
+    * ``name``: (required) value of type string.
```

Comment:
Is "value of type string" right here?  Same as your original version above; seems like this should be about the name field.

Reply:
> Is "value of type string" right here?

 What was meant here is that the value of the name member should be a string. I'll replace it just by string